core/src/components/accordion/test/multiple/accordion.e2e.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('accordion: multiple', () => {
+  test('should update value and not have visual regressions', async ({ page }) => {
+    await page.goto(`/src/components/accordion/test/multiple`);
+    const accordionGroup = page.locator('ion-accordion-group');
+    const diningHeader = page.locator('ion-accordion[value="dining"] ion-item[slot="header"]');
+    const attractionsHeader = page.locator('ion-accordion[value="attractions"] ion-item[slot="header"]');
+
+    // toHaveValue doesn't work with non-input elements
+    await expect(accordionGroup).toHaveAttribute('value', 'attractions');
```

Comment:
Optional: You can also do `toHaveJSProperty` if you want to check the actual JS property value